packages/ember-metal/tests/weak_map_test.js

```diff
@@ -53,6 +53,26 @@ QUnit.test('that error is thrown when using a non object key', function(assert)
   throws(function() {
     map.set('a', 1);
   }, /Uncaught TypeError: Invalid value used as weak map key/);
+
+  throws(function() {
```

Comment:
assertion testing should use `expectAssertion` (`assert` does not throw in prod builds so `expectAssertion` handles that for us).